docs/examples/clusters/glusterfs/userdata/bootstrap

```diff
@@ -0,0 +1,152 @@
+		sleep 20
+        gluster volume create glustervol1 replica 3 transport tcp $host3:/bricks/brick3/brick $server2:/bricks/brick2/brick $server1:/bricks/brick1/brick force
+		sleep 10
+        gluster volume start glustervol1
+		;;
+	"glusterfs-client1")
+		sed -i '/search/d' /etc/resolv.conf 
+        echo "search baremetal.oraclevcn.com publicsubnetad2.baremetal.oraclevcn.com publicsubnetad1.baremetal.oraclevcn.com publicsubnetad3.baremetal.oraclevcn.com localdomain" >> /etc/resolv.conf
+        chattr -R +i /etc/resolv.conf
+		#firewall-cmd --zone=public --add-port=111/tcp --add-port=139/tcp --add-port=445/tcp --add-port=965/tcp --add-port=2049/tcp \
+		#--add-port=38465-38469/tcp --add-port=631/tcp --add-port=111/udp --add-port=963/udp --add-port=49152-49251/tcp  --permanent
+		#firewall-cmd --reload
+		systemctl disable firewalld
+        systemctl stop firewalld
+		yum install glusterfs glusterfs-fuse attr -y
+		mkdir /mnt/glustervol1
+		sleep 4m
+        echo "glusterfs-server1:/glustervol1 /mnt/glustervol1	glusterfs defaults,_netdev  0 0" >> /etc/fstab
+		mount -a
+		;;
+	"glusterfs-client2")
+		sed -i '/search/d' /etc/resolv.conf
+        echo "search baremetal.oraclevcn.com publicsubnetad2.baremetal.oraclevcn.com publicsubnetad1.baremetal.oraclevcn.com publicsubnetad3.baremetal.oraclevcn.com localdomain" >> /etc/resolv.conf
+        chattr -R +i /etc/resolv.conf
+        #firewall-cmd --zone=public --add-port=111/tcp --add-port=139/tcp --add-port=445/tcp --add-port=965/tcp --add-port=2049/tcp \
+        #--add-port=38465-38469/tcp --add-port=631/tcp --add-port=111/udp --add-port=963/udp --add-port=49152-49251/tcp  --permanent
+        #firewall-cmd --reload
+		systemctl disable firewalld
+        systemctl stop firewalld
+        yum install glusterfs glusterfs-fuse attr -y
+        mkdir /mnt/glustervol1
+        sleep 4m
+        echo "glusterfs-server2:/glustervol1 /mnt/glustervol1   glusterfs defaults,_netdev  0 0" >> /etc/fstab
+        mount -a
+        ;;
+	"glusterfs-client3")
+		sed -i '/search/d' /etc/resolv.conf
+        echo "search baremetal.oraclevcn.com publicsubnetad2.baremetal.oraclevcn.com publicsubnetad1.baremetal.oraclevcn.com publicsubnetad3.baremetal.oraclevcn.com localdomain" >> /etc/resolv.conf
+        chattr -R +i /etc/resolv.conf
+        #firewall-cmd --zone=public --add-port=111/tcp --add-port=139/tcp --add-port=445/tcp --add-port=965/tcp --add-port=2049/tcp \
+        #--add-port=38465-38469/tcp --add-port=631/tcp --add-port=111/udp --add-port=963/udp --add-port=49152-49251/tcp  --permanent
+        #firewall-cmd --reload
+		systemctl disable firewalld
+        systemctl stop firewalld
+        yum install glusterfs glusterfs-fuse attr -y
+        mkdir /mnt/glustervol1
+        sleep 4m
+        echo "glusterfs-server3:/glustervol1 /mnt/glustervol1   glusterfs defaults,_netdev  0 0" >> /etc/fstab
+        mount -a
+        ;;
+	*)
+        echo "$glusterfshost" >>/tmp/notfound.txt
+		;;
+esac
```

Comment:
I would recommend not publishing this example. This code looks hard to test for correctness, it looks hard to decide if it is minimalistic (or does more stuff than what is needed), hard to decide if it's secure etc.

I would assume, if published by somebody from Oracle, that this copy will be copy-pasted.